controller/space_iterations_test.go

```diff
@@ -131,8 +135,11 @@ func (rest *TestSpaceIterationREST) TestSuccessCreateIterationWithOptionalValues
 	ci := createSpaceIteration(iterationName, &iterationDesc)
 	application.Transactional(rest.db, func(app application.Application) error {
 		repo := app.Spaces()
+		spaceTemplate, err := spacetemplate.CreateEmptySpaceTemplateWithRepo(context.Background(), app.SpaceTemplates())
+		require.Nil(rest.T(), err)
 		testSpace := space.Space{
-			Name: "TestSuccessCreateIterationWithOptionalValues-" + uuid.NewV4().String(),
+			Name:            "TestSuccessCreateIterationWithOptionalValues-" + uuid.NewV4().String(),
```

Comment:
I created a function to set the test name satisfying the naming restrictions. It is in common.go in test package